etc/env/docker.env

```diff
@@ -2,10 +2,12 @@ ETH_CLIENT_WEB3_URL=http://rskj:4444
 FEE_TICKER_COINMARKETCAP_BASE_URL=http://dev-ticker:9876
 FEE_TICKER_COINGECKO_BASE_URL=http://dev-ticker:9876
 DATABASE_URL=postgres://postgres@postgres/plasma
-FEE_TICKER_UNISWAP_URL=http://dev-liquidity-token-watcher:9975/graphql
 DEV_LIQUIDITY_TOKEN_WATCHER_BLACKLISTED_TOKENS=0x0000000000000000000000000000000000000001
 DEV_LIQUIDITY_TOKEN_WATCHER_DEFAULT_VOLUME=500
 DEV_LIQUIDITY_TOKEN_WATCHER_REGIME=whitelist
+ETH_CLIENT_CHAIN_ID=33
+ETH_CLIENT_GAS_PRICE_FACTOR=1
+ETH_CLIENT_WEB3_URL=http://rskj:4444
```

Comment:
This value is present twice, see line #1

Reply:
done